### ISSUE TITLE
FGD parsing bug: FGD format is whitespace agnostic

### ISSUE THREAD

**Issue by mfaizsyahmi** [UnrealKaraulov/newbspguy#125]
FGD format is whitespace agnostic. In reference implementation i.e. Hammer 3.x, as well as in JACK, you can spread a Class definition across several lines like so:
```fgd
@PointClass 
	base(Target, Targetname, Light, ZHLT_point) 
	flags(Light) 
	iconsprite("sprites/light.spr") 
	= light : "Invisible lightsource." : "https://twhl.info/wiki/page/light"
[
	// entity attributes here
]
```
and it is valid FGD.

For some reason, bspguy's FGD parser doesn't do this, expecting the entire class definition to be on a single line.

**Comment by UnrealKaraulov** [comment]
@mfaizsyahmi yes... newbspguy fgd parser is very poor, and need fully rewrite, but I no has time at this moment, you can use normal fgds without spaces and newlines in class definitions.
